examples/credential_issuer_metadata_jwt_vc_json.json

```diff
@@ -34,38 +34,43 @@
       "credential_signing_alg_values_supported": [
         "ES256"
       ],
-      "credential_definition": {
+      "credential_definition":{
```

Comment:
Isn't `@contex` required?

Reply:
we used to have `@context` but previously had that discussion before and agreed to remove it